atlas/rules/genecatalog.snakefile

```diff
@@ -463,12 +463,12 @@ rule generate_subsets_for_annotation:
     output:
         temp(dynamic("Genecatalog/subsets/genes/{subsetID}.faa"))
     params:
-        subset_size=config['genecatalog']['SubsetSize'],
-        output_dir= lambda wc, output: os.path.dirname(output[0]),
+        subset_size = config['genecatalog']['SubsetSize'],
+        output_dir = lambda wc, output: os.path.dirname(output[0]),
     run:
-        from utils import fasta
+        from scripts.utils import fasta
```

Comment:
line 8 in the Snakemake should add the scripts dir to the python path. So this shouldn't be necessary.

Reply:
For some reason, I got an error that the name 'fasta' could not be found, when this line of code ran. I wonder if it is because of a name collision -- 'utils.py' is also the name of a script in the main atlas folder.